src/vs/platform/clipboard/common/clipboardService.ts

```diff
@@ -53,4 +53,6 @@ export interface IClipboardService {
 	 * Used for implementations such as web which do not always support using the real clipboard.
 	 */
 	clearInternalState?(): void;
+
+	readImage(): Promise<Uint8Array>;
```

Comment:
Some jsdoc would be helpful here. Would be handy to mention if this causes a permissions dialog on web.